data/human/ships.txt

```diff
@@ -1958,11 +1958,11 @@ ship "Gunboat"
 		"Heavy Laser Turret"
 		
 		"RT-I Radiothermal"
-		"LP144a Battery Pack"
+		"LP072a Battery Pack"
 		"D14-RN Shield Generator"
 		"Small Radar Jammer"
-		"Cargo Scanner"
-		"Outfit Scanner"
+		"Cargo Scanner" 9
+		"Outfit Scanner" 7
```

Comment:
```suggestion
		"Cargo Scanner" 8
		"Outfit Scanner" 3
```

This brings the cargo scan range from 900 down to 848 and outfit scan range from 1322 down to 866.

This will also increase the time to do a cargo scan by a small amount; and significantly increase the time to do an outfit scan.

Reply:
Okay, you make good points. I, amazingly, hadn't considered the "ambushed while actually smuggling illicit cargo" perspective. I'll accept that, and cover the other scanning ships I've been made aware of.

(It should be noted, however, that pre-scan-change, the longest a scanner would take is 60 frames, one second. By comparison, 3 seconds is an improvement, even if only a modest one.)

Reply:
I will also increase tactical scanner count by 1, so that the powers are 76, 75, and 64, which gives similar ranges to all three of them (848.5, 866.02, and 800 flat).

Reply:
Sure, bumping up the tactical scanner by 1 seems reasonable.